demo/demo.ts

```diff
@@ -48,7 +47,7 @@ import {
            </div>
         </div>
         <div class="col-md-4">
-          <h3>{{ date | calendarTitle:view }}</h3>
+          <h3>{{ date | calendarDate:'dayViewTitle' }}</h3>
```

Comment:
Sure! It's because the title is for the month, week and day views - each view has their own style of title. Currently you're hard coding it to the day view title so it will only ever show that style of title.